grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py

```diff
@@ -491,7 +499,32 @@ def create_register_file(self, filename):
                         br='S2_{}'.format(band_ref)
                     ))
                 fd.write(os.linesep)
+
+    def set_footprint(self,fp):
+        reg_name = "tmpregion_%s"%(str(os.getpid()))
+        gs.run_command('g.region',save=reg_name)
+        fp_dict={}
+        idlist = [x for x in gs.parse_command('v.db.select',map=fp)]
```

Comment:
If the map identifyer could be used in a where clause, this would not be necessary, cause `where="identifyer = {}".format(map_identifyer)` could be added to the `v.db.select` call for getting the footprint region as well as to `v.to.rast` 
```

Reply:
Thanks, this now included as well